package.json

```diff
@@ -3,26 +3,58 @@
   "version": "0.12.0",
   "private": true,
   "scripts": {
+    "bazel": "bazelisk",
+    "bazelrc:ci": "cp ./.github/.bazelrc /etc/bazel.bazelrc",
+    "bazelrc": "tools/generate-bazelrc.sh",
+    "bazel:buildifier": "find . -type f \\( -name \"*.bzl\" -or -name WORKSPACE -or -name BUILD -or -name BUILD.bazel \\) ! -path \"*/node_modules/*\" | xargs buildifier -v --warnings=attr-cfg,attr-license,attr-non-empty,attr-output-default,attr-single-file,constant-glob,ctx-args,depset-iteration,depset-union,dict-concatenation,duplicated-name,filetype,git-repository,http-archive,integer-division,load,load-on-top,native-build,native-package,output-group,package-name,package-on-top,redefined-variable,repository-name,same-origin-load,string-iteration,unused-variable,unsorted-dict-items,out-of-order-load",
```

Comment:
wtf?

Reply:
This is basically one giant lint rule for Bazel BUILD/WORKSPACE files. I agree it's lengthy, and I'll talk to the team about finding a way to consolidate it, but it works well and covers all of the basic needs of a linter.

Reply:
can't we put it in under `tools/bazel-buildiufer.sh` or something?